src/igraph/io/objects.py

```diff
@@ -249,15 +249,15 @@ def _construct_graph_from_list_dict(
     multiple values y: the edge (x,y) will be created in the graph. x and y
     must be either one of:
 
-    - two integers: the vertices with those ids will be connected
-    - two strings: the vertices with those names will be connected
+      - two integers: the vertices with those ids will be connected
+      - two strings: the vertices with those names will be connected
 
     If names are used, the order of vertices is not guaranteed, and each
     vertex will be given the vertex_name_attr attribute.
 
     @param edges: the dict of sequences describing the edges
-    @param directed (bool): whether to create a directed graph
-    @vertex_name_attr (str): vertex attribute that will store the names
+    @param directed: (bool) whether to create a directed graph
+    @vertex_name_attr: (str) vertex attribute that will store the names
```

Comment:
Type annotations can be removed, or moved to the function headers now that these are ubiquitous in Python.

Reply:
yes